esp/public/media/scripts/onsite/teacher_checkin.js

```diff
@@ -145,17 +145,17 @@ $j(function(){
         textTeacher($j(this))
     });
 
-    var skip_semi_checked_in = false
-    $j("#skip-semi-checked-in").click(function() {
-        skip_semi_checked_in = !skip_semi_checked_in
+    var skip_semi_checked_in_teachers = false
+    $j("#skip-semi-checked-in-teachers").click(function() {
+        skip_semi_checked_in_teachers = !skip_semi_checked_in_teachers
```

Comment:
I know I wrote the original, but now I'm wondering if there's any way this could get out of sync with the state of the checkbox? Should we get the checked status of the checkbox instead of just alternating each time it gets changed?